packages/instantsearch.js/src/components/Template/Template.tsx

```diff
@@ -110,7 +110,11 @@ See: https://www.algolia.com/doc/guides/building-search-ui/upgrade-guides/js/#up
 
     // This is to handle Hogan templates with Fragment as rootTagName
     if (RootTagName === Fragment) {
-      return <RawHtml content={content} key={content} />;
+      return content.trim().charAt(0) === '<' ? (
+        <RawHtml content={content} key={content} />
```

Comment:
I'd like to have a test of `template` with a fragment root and content like `hello {{world}}` to ensure hogan is still transformed first.

Reply:
Done !